exercisePPJ_06_I/FunctionPPJ_06_I.java

```diff
@@ -0,0 +1,52 @@
+public class FunctionPPJ_06_I {
+	
+	private double binaryWrt[] = {0,0,0,0,0,0,0,0};
+	private double wrtDouble = 0;
+	
+	public void setFunction(double wrtD) {
+		wrtDouble = wrtD;
+	}
+	
+	public void getFunction() {
+		
+		for(int counter = 7; counter >= 0; counter--) {
+		
+		int exponent = 0;
+		
+		while(wrtDouble > Math.pow(2,exponent)) {
+			exponent++;
+		}
```

Comment:
Read above `while` loop.
Try to think what it does.
If you know, please, explain to me:)

Reply:
@MatDls I don't know what do you mean. this construction working good